migrate/sql/20180103142001_initial_schema.sql

```diff
@@ -19,27 +19,27 @@ CREATE TABLE IF NOT EXISTS users (
     PRIMARY KEY (id),
 
     id            UUID          NOT NULL,
-    username      VARCHAR(128)  CONSTRAINT users_username_key UNIQUE NOT NULL,
+    username      VARCHAR(128)  NOT NULL CONSTRAINT users_username_key UNIQUE,
     display_name  VARCHAR(255),
     avatar_url    VARCHAR(512),
     -- https://tools.ietf.org/html/bcp47
-    lang_tag      VARCHAR(18)   DEFAULT 'en',
+    lang_tag      VARCHAR(18)   NOT NULL DEFAULT 'en',
```

Comment:
I added a `NOT NULL` clause because I can't see why it wouldn't be backwards compatible and appears to have been forgotten in the original commit.